sourmash

```diff
@@ -1,5 +0,0 @@
-#! /usr/bin/env python
-# NOTE: this script is only used in 'developer' mode (python setup.py
-# build_ext -i) -- see setup.py entrypoints for what end-users use.
-from sourmash_lib.__main__ import main
-main()
```

Comment:
Is there no command line entry point now? Or should we do something like `pip install -e .` to set up a dev environment?

Reply:
You need `pip install -e .` (or `python setup.py develop`) to have the `sourmash` command available (it uses the entry point defined in `setup.py`).